vignettes/SEIRD_CT.Rmd

```diff
@@ -0,0 +1,362 @@
+
+## Basic reproduction number
+It can be shown that the basic reproduction number for this model is given by^[Following [@vandenDriessche17] (see `Vignette("SEIaImIsRD")` for more information) and assuming a state order: $(E, P, A, I, E^T, P^T, A^T, I^T)$, we obtain the matrices:
+$$
+F = \left(\begin{array}{cc} 
+0 & \beta (1-\pi) & \beta_a & \beta (1-\pi) & 0 & 0 & 0 & 0\\
+0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+0 & \beta \pi & 0 & \beta \pi & 0 & 0 & 0 & 0\\
+0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+0 & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+0 & 0 & 0 & 0 & 0 & 0 & 0 & 0
+\end{array}\right),
+$$
+and
+$$
+V = \left(\begin{array}{cc} 
+\omega & 0 & 0 & 0 & 0 & 0 & 0 & 0\\
+-(1-\eta_a)\omega & \psi & 0 & 0 & 0 & 0 & 0 & 0\\
+-\eta_a\omega & 0 & \gamma & 0 & 0 & 0 & 0 & 0\\
+0 & -(1-\phi)\psi & 0 & \gamma+\mu & 0 & 0 & 0 & 0\\
+0 & 0 & 0 & 0 & \omega & 0 & 0 & 0\\
+0 & 0 & 0 & 0 & -(1-\eta_a)\omega & \psi & 0 & 0\\
+0 & 0 & 0 & 0 & -\eta_a\omega & 0 & \gamma & 0\\
+0 & -\phi\psi & 0 & 0 & 0 & -\psi & 0 & \gamma+\mu
+\end{array}\right)
+$$
+The matrix $K=FV^{-1}$ is called the next generation matrix and the maximum absolute eigenvalue of it yields the basic reproduction number given in the main text.
+]:
+
+\begin{equation}
+R_0 = \eta_a\underbrace{\frac{\beta_a}{\gamma_a}}_{\text{asymptomatic cases}} + (1 - \eta_a)(1 - \pi) \left[\underbrace{\frac{\beta}{\psi}}_{\text{presymptomatic cases}} + (1-\phi) \underbrace{\frac{\beta}{\gamma+\mu}}_{\text{symptomatic cases}}\right].
+\end{equation}
+
+As the underbraces show in the equation above, this expression has ample intuition behind it. The average number of cases generated from a given infection equates to the sum of: the average cases generated if the infection were asymptomatic, weighted by the probability of a case being asymptomatic; the average number of cases from presymptomatics which are the overall cases which are not asymptomatic $(1-\eta_a)$ which are not quarantined due to contact tracing $(1-\pi)$; and those generated from symptomatics for the same overall fraction of cases $(1-\eta_a)(1-\pi)$ and who do not self isolate when their symptoms start $(1-\phi)$.
+
+The expression for $R_0$ shows that even if contact tracing and isolation of contacts is perfect ($\pi=1$), there remains transmission due solely to asymptomatic cases. A meta-analysis concluded that the overall proportion of cases due to asymptomatics is 17% [@byambasuren2020estimating], meaning that if the transmission rate of asymptomaics, $\beta_a$ is high enough (or, although this model does not include it, a longer duration of infection), it is possible to sustain an epidemic solely through asymptomatic transmission: if $\eta \beta_a/\gamma>1$. This analysis suggests that regular testing, regardless of whether an individual has symptoms, may be an effective way to target asymptomatic reservoirs of infection, if upon testing positive individuals and their close contacts isolate.
```

Comment:
Yes, contact tracing can find asymptomatic people (i.e. contacts of symptomatics). But contact tracing is unlikely to find contacts (symptomatic or otherwise) of asymptomatic people because so few asymptomatics are identified.